Blockzilla/PhotoManager.swift

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+protocol PhotoManagerDelegate: class {
+    func photoManager(_ photoManager: PhotoManager, didFinishSavingWithError error: Error?)
+}
+
+class PhotoManager: NSObject {
+    weak var delegate: PhotoManagerDelegate?
+
+    func save(image: UIImage) {
```

Comment:
Cool so please update https://github.com/mozilla-mobile/focus-ios/issues/123 with what QA should expect if i) the user has previously disabled photo access and ii) the user hasn't been prompted yet, just so QA knows that the expected behavior is. Screenshots are nice but verbiage is fine too.